tests/by-util/test_echo.rs

```diff
@@ -252,6 +252,24 @@ fn test_double_hyphens() {
         .stdout_only("-- --\n");
 }
 
+#[test]
+fn test_double_hyphens_between() {
+    new_ucmd!()
+        .arg("a")
+        .arg("--")
+        .arg("b")
+        .succeeds()
+        .stdout_only("a -- b\n");
+
+    new_ucmd!()
+        .arg("foo ")
+        .arg("-- --")
+        .arg("bar")
+        .arg("hehehe")
```

Comment:
A detail: can you please add "hehehe" to the "spell-checker ignore" line at the top of the file to fix https://github.com/uutils/coreutils/actions/runs/14036814735/job/39296750477?pr=7559?

Reply:
> A detail: can you please add "hehehe" to the "spell-checker ignore" line at the top of the file, to fix https://github.com/uutils/coreutils/actions/runs/14036814735/job/39296750477?pr=7559?

Okay, done.